logger/commonConverters.go

```diff
@@ -0,0 +1,29 @@
+package logger
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+const msgFixedLength = 40
+
+func displayTime(time time.Time) string {
+	return fmt.Sprintf("%d-%02d-%02d %02d:%02d:%02d",
```

Comment:
I think this is hard to follow. I recommend this approach:
`return time.Format("2006-01-02 15:04:05")` 
it will return the same stuff and it's easier to change.
Also rename `time` parameter as it collides with a package name

Reply:
Done